__tests__/app/renderer/src/components/__snapshots__/ResultItem.js.snap

```diff
@@ -1,6 +1,6 @@
 exports[`<ResultItem /> component should render a result item component in a selected state 1`] = `
 <li
-  data-css-1j2x6j7=""
+  data-css-fsydme=""
```

Comment:
@vutran I'm not too experienced with the way Jest works with snapshots.
So after reading the doc and since my changes were legit I updated the snapshot to match the new output. Is there anything I should do?